src/api/maths/shared/elapsedTime.lua

```diff
@@ -0,0 +1,19 @@
+---elapsedTime
+---@param firstTime number
+---@param secondTime number
+---@param format boolean
+---@return number or table
+---@public
+function API_Maths:elapsedTime(firstTime, secondTime, format)
+    if (not firstTime or not secondTime) then
+        return
```

Comment:
it would be logical to return 0 wouldn't it?
```suggestion
        return 0
```